app/services/git/fetches_tracks.rb

```diff
@@ -1,25 +1,26 @@
 class Git::FetchesTracks
 
-  QUIET_PERIOD = 30.seconds
   ERROR_BACKOFF_PERIOD = 10.seconds
 
-  def self.run
-    new.run
+  def self.fetch(tracks)
+    new(tracks).fetch
   end
 
-  def run
-    loop do
-      fetch_problem_specs
-      fetch_website_content
-      Track.find_each do |track|
-        fetch(track)
-        sleep 1.second
-      end
-      sleep QUIET_PERIOD
+  def initialize(tracks)
+    @tracks = tracks
+  end
+
+  def fetch
+    fetch_problem_specs
+    fetch_website_content
```

Comment:
We need to address this two fetches, because they're not tracks, but are specialisations of Git::RepoBase and therefore need to be fetched and triggered by a webhook - sorry for the scope creep!

Reply:
I think we can stick with this code now and just generalize to `GitUpdateFetches` :)

Reply:
I don't think so - what if just the website content changes? That should trigger a webhook which triggers a refresh of that content.

Reply:
Good point. I'd need to write some code to account for the problem specs and website copy repos changing.